plugin/core/panels.py

```diff
@@ -191,3 +191,15 @@ def run(self, edit: sublime.Edit, window_id: int) -> None:
                 if not erase_region.empty():
                     self.view.erase(edit, erase_region)
         clear_undo_stack(self.view)
+
+
+class LspClearLogPanelCommand(sublime_plugin.WindowCommand):
+    def run(self) -> None:
+        if is_log_panel_open(self.window):
+            panel = ensure_server_panel(self.window)
+            if not panel:
+                return
+            panel.run_command("lsp_clear_panel")
+
+    def is_visible(self) -> bool:
+        return is_log_panel_open(self.window)
```

Comment:
With it being Window Command the menu item appears in every view as long as log panel is open which doesn't sound right.

Is it possible to make it a TextCommand and restrict to the log panel view?

Reply:
Even if TextCommand is not an option (not sure how things work with menu items) then we could still check what is the active view here.